src/dsp_tools/resources/start-stack/docker-compose.yml

```diff
@@ -45,6 +45,8 @@ services:
     # on the verge of every deployment (fortnightly), update the "image" value from the "app" value of
     # https://github.com/dasch-swiss/ops-deploy/blob/main/versions/RELEASE.json
     image: daschswiss/dsp-app:v11.22.1
+    volumes:
+      - ./dsp-app-config.json:/public/config/config.prod.json
```

Comment:
This is not clean, but that's the way the app works, so we have to do it the same way.

Reply:
I was not so clear in my description. I have updated it. Is it now better understandable?

Short answer: I just copied [this file](https://github.com/dasch-swiss/dsp-das/blob/eca422202fd5653569b6cff2abdf6a327aa2b49f/apps/dsp-app/src/config/config.prod.json), and set `allowEraseProjects` to True.